examples/files.json

```diff
@@ -325,7 +325,8 @@
 	"webgpu": [
 		"webgpu_sandbox",
 		"webgpu_rtt",
-		"webgpu_compute"
+		"webgpu_compute",
+		"webgpu_nodes"
```

Comment:
I would rename it to `webgpu_materials`. Node materials will be the default so there is no need to explicitly name it.

As a sidenote: I recommend to avoid making this example as complex as `webgl_materials_nodes`. I had a really hard time in the past to apply changes to it^^.

Reply:
I would like to use this example to test the `nodes` like `NormalNode`, `PositionNode`, etc. You think about keeping a example for this?

Reply:
That's of course okay as long as we don't make it too complex^^.